Mage/src/main/java/mage/players/PlayerImpl.java

```diff
@@ -4771,6 +4771,11 @@ public SpellAbility chooseAbilityForCast(Card card, Game game, boolean noMana) {
         return card.getSpellAbility();
     }
 
+    @Override
+    public SpellAbility chooseAbilityForCast(Card card, List<SpellAbility> spellAbilities, Game game) {
+        return spellAbilities.get(0);
```

Comment:
Must check on size, e.g. stream.first

Reply:
it can't be empty so it shouldn't have to check